CITATION.cff

```diff
@@ -0,0 +1,1566 @@
+        VA 22903, USA
+    - family-names: Yoachim
+      given-names: Peter
+      orcid: https://orcid.org/0000-0003-2874-6464
+      affiliation: Department of Astronomy, University of Washington, Seattle, WA
+        98195, USA
+    - family-names: ZHANG
+      given-names: Chen
+      orcid: https://orcid.org/0000-0002-9583-263X
+      affiliation: Purple Mountain Observatory, CAS, Qixia District, Nanjing 210023,
+        China
+    - family-names: Zonca
+      given-names: Andrea
+      orcid: https://orcid.org/0000-0001-6841-1058
+      affiliation: San Diego Supercomputer Center, University of California San Diego,
+        9500 Gilman Dr, La Jolla, CA 92093, USA
+  abstract: >-
+    The Astropy Project supports and fosters the development of
+    open-source and openly developed Python packages that provide
+    commonly needed functionality to the astronomical community. A key
+    element of the Astropy Project is the core package astropy, which
+    serves as the foundation for more specialized projects and
+    packages. In this article, we summarize key features in the core
+    package as of the recent major release, version 5.0, and provide
+    major updates on the Project. We then discuss supporting a broader
+    ecosystem of interoperable packages, including connections with
+    several astronomical observatories and missions. We also revisit
+    the future outlook of the Astropy Project and the current status
+    of Learn Astropy.  We conclude by raising and discussing the
+    current and future challenges facing the Project.
+  keywords:
+    - Astronomy software
+    - Open source software
+    - Astronomy data analysis
+    - "Astrophysics - Instrumentation and Methods for Astrophysics"
+  type: article
+  journal: "The Astrophysical Journal"
+  volume: 935
+  issue: 2
+  pages: 20
+  year: 2022
+  identifiers:
+    - description: ADS bibcode
+      type: url
+      value: "https://ui.adsabs.harvard.edu/abs/2022ApJ...935..167A"
+    - description: The Astrophysical Journal
+      type: doi
+      value: 10.3847/1538-4357/ac7c74
+    - description: arXiv
+      type: doi
+      value: 10.48550/arXiv.2206.14220
+    - description: Publisher PDF (open access)
+      type: url
+      value: "https://iopscience.iop.org/article/10.3847/1538-4357/ac7c74/pdf"
+    - description: ArXiv eprint (open access)
+      type: url
+      value: "https://arxiv.org/pdf/2206.14220.pdf"
+
+references:
+  - title: >-
+      The Astropy Project: Building an Open-science Project and Status
+      of the v2.0 Core Package
+    authors:
+      - name: "Astropy Collaboration"
+      - family-names: Price-Whelan
+        given-names: A. M.
+        orcid: https://orcid.org/0000-0003-0872-7098
+        affiliation: Department of Astrophysical Sciences, Princeton University, Princeton,
+          NJ 08544, USA
+      - family-names: Sipőcz
+        given-names: B. M.
+        orcid: https://orcid.org/0000-0002-3713-6337
```

Comment:
I think @bsipocz's affiliation got dropped here. Based on the published paper, it should read:
```suggestion
      - family-names: Sipőcz
        given-names: B. M.
        orcid: https://orcid.org/0000-0002-3713-6337
        affiliation: Institute of Astronomy, University of Cambridge, Madingley Road, Cambridge, CB3 0HA, UK
```

Reply:
That is correct. I took the repository for the affiliation, and [that had](https://github.com/astropy/astropy-v2.0-paper/blob/c31a57c6ffcf6b040131c15b472d85d9ecdbcc3e/author.tex#L25-L26):
```LaTeX
\author[0000-0002-3713-6337]{B. M. Sip\H{o}cz}
\noaffiliation
```
@